### PR TITLE
feat: add checkbox list

### PR DESCRIPTION
**Pull request by hugoattal**
### Description

Adding checkbox list to controls.

Example:

image

### What is the purpose of this pull request? 

- [ ] Bug fix
- [x] New Feature
- [ ] Documentation update
- [ ] Other

### CODE REVIEW

File: packages/histoire-controls/histoire.config.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { defineConfig } from 'histoire'
```
Comment (Akryum) [inline packages/histoire-controls/histoire.config.ts:1]: The book moved to `histoire-controls-stories`

File: packages/histoire-controls/package.json [hunk 2]
```diff
@@ -34,6 +34,7 @@
     "build": "rimraf dist && pnpm run build:css && vite build && pnpm run build:types",
     "build:css": "tailwindcss -i src/style/main.css -o ./dist/style-standalone.css -c tailwind.config.cjs --postcss postcss.config.cjs",
     "build:types": "vue-tsc --declaration --emitDeclarationOnly",
+    "story:dev": "histoire dev",
```
Comment (Akryum) [inline packages/histoire-controls/package.json:37]: Revert this file

File: pnpm-lock.yaml [hunk 3]
```diff
@@ -378,6 +378,7 @@ importers:
 
   packages/histoire-controls:
```
Comment (Akryum) [inline pnpm-lock.yaml:379]: Revert this file

**Review by Akryum — CHANGES_REQUESTED** [review]
Nice job!
- Don't put histoire as a (dev)dependency of histoire-controls otherwise we end up with a cyclic dependency. Instead the controls stories are served by histoire-controls-stories.
- Svelte types for the new builtin components are missing in the Svelte plugin
https://github.com/histoire-dev/histoire/blob/84bb3da4c3fffbe460e245aab27ead0b495f3c18/packages/histoire-plugin-svelte/src/index.ts#L55

File: packages/histoire-controls/histoire.config.ts (hunk 1 above)
Comment (hugoattal) [reply packages/histoire-controls/histoire.config.ts]: Should have seen it *facepalm* 😅... Removed here: https://github.com/histoire-dev/histoire/pull/310/commits/67c61e29a3b9e75c71f4a06c2ae1e29712259a99

File: packages/histoire-controls/package.json (hunk 2 above)
Comment (hugoattal) [reply packages/histoire-controls/package.json]: Done here: https://github.com/histoire-dev/histoire/pull/310/commits/67c61e29a3b9e75c71f4a06c2ae1e29712259a99

File: pnpm-lock.yaml (hunk 3 above)
Comment (hugoattal) [reply pnpm-lock.yaml]: Done here: https://github.com/histoire-dev/histoire/pull/310/commits/67c61e29a3b9e75c71f4a06c2ae1e29712259a99

**Review by Akryum — CHANGES_REQUESTED** [review]
The new component is missing from the `index.ts` (export + `components`)

**Comment by Akryum** [comment]
https://github.com/histoire-dev/histoire/blob/main/packages/histoire-controls/src/index.ts

**Comment by alvarosabu** [comment]
Looks good

**Pull request merged**